src/gc/gc.cpp

```diff
@@ -7035,7 +7033,7 @@ uint32_t* gc_heap::make_card_table (uint8_t* start, uint8_t* end)
     card_table_lowest_address (ct) = start;
     card_table_highest_address (ct) = end;
     card_table_brick_table (ct) = (short*)((uint8_t*)ct + cs);
-    card_table_size (ct) = alloc_size_aligned;
```

Comment:
>card_table_size [](start = 4, length = 15)

if you remove the size from the VirtualRelease call, card_table_size can also be removed.